ci/glci/model.py

```diff
@@ -272,6 +272,14 @@ class GcpPublishedImage(PublishedImageBase):
     gcp_project_name: str
 
 
+@dataclasses.dataclass(frozen=True)
+class AzurePublishedImage:
+    transport_state: str
+    urn: str
```

Comment:
I know, the other dataclasses are also very sparse with documentation.

however, could you add a brief doc-str explaining those attrs (e.g. what the urn refers to, which transport_states there are..); also, it might be nice to use an enum for TransportState

Reply:
Added some doc-str and add an enum class `glci.model.AzureTransportState`.